src/lib/CuratorCard.svelte

```diff
@@ -19,7 +19,7 @@
       >
       <p class="roles">
         {(roles.seniorCurator ? 'Senior Curator' : 'Curator') +
-          (roles.verifiedMapper ? ', Mapper' : '')}
+          (roles.verifiedMapper ? ', Verified Mapper' : '')}
```

Comment:
This will just be plain wrong for some curators. It seems you are aware of that, thoug.
Are we potentially offending someone or somegroup with that?

Reply:
This point was wrong. It's actually correct (as you showed me in private chat). I had remembered the issue wrong.